### ISSUE TITLE
Feature request: self-centered formatting

### ISSUE THREAD

**Issue by not-AMM** [mb21/panwriter#102]
Right now, Panwriter offers formatting after selecting the text you want to format, but it would be cool to also offer formatting before selecting. I mean, you press `ctrl + b` and the `****[cursor]` appears, but instead, the cursor moves to the center `**[cursor]**` and then you write.

Panwriter is becoming one of my favourite editors!

**Comment by mb21** [comment]
agreed, that would be nice. haven't time right now, but would probably be a fairly simple change around here: https://github.com/mb21/panwriter/blob/master/src/components/Editor/Editor.tsx#L88
